doc/source/user_guide/indexing.rst

```diff
@@ -676,7 +676,7 @@ Current behavior
    KeyError in the future, you can use .reindex() as an alternative.
 
    See the documentation here:
-   https://pandas.pydata.org/pandas-docs/stable/indexing.html#deprecate-loc-reindex-listlike
+   :ref:`Deprecation doc <indexing.deprecate_loc_reindex_listlike>`
```

Comment:
You can make the "here" the text of the reference (and remove the colon), I think.